invenio_app_ils/document_requests/views.py

```diff
@@ -97,7 +97,7 @@ class PendingRequestResource(DocumentRequestActionResource):
     @need_permissions("document-request-pending")
     def post(self, pid, record, **kwargs):
         """Pending request post method."""
-        # expecting remove_fields list of fields to remove
+        # NOTE: remove_fields is a list of props to remove from the record
```

Comment:
We need to discuss this IRL! :) I am not sure what exactly is this `remove_fields`.